src/odemis/acq/align/test/align_test.py

```diff
@@ -48,152 +44,140 @@ def setUp(self):
+        self.img_spccd_lon = img.ensure2DImage(self.data[0])
+
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-off-slit-ccd.ome.tiff"))
+        self.img_ccd_loff = img.ensure2DImage(self.data[0])
+
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-on-slit-ccd.ome.tiff"))
+        self.img_ccd_lon = img.ensure2DImage(self.data[0])
 
-    def test_slit_off_on(self):
+    def test_slit_off_on_spccd(self):
         """
         Test from dark slit image to bright slit image (gray)
         """
         bl = self.light
-        ccd = mock.FakeCCD(0, self.img_light_off)
-        f = turnLightAndCheck(bl, ccd)
+        ccd = mock.FakeCCD(self.img_spccd_loff)
+        f = turnOnLightAndCheck(bl, ccd)
         time.sleep(1)
         self.assertFalse(f.done())
 
         # simulate light turning on
-        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-on-slit-spccd-simple.ome.tiff"))
-        img_light_on = img.ensure2DImage(self.data[0])
-
-        ccd.fake_img = img_light_on
-        time.sleep(1)
+        ccd.fake_img = self.img_spccd_lon
+        time.sleep(3)
         self.assertTrue(f.done())
         f.result()
 
- #    @skip("skip")
-    def test_slit_timeout(self):
+
+    def test_slit_off_on_ccd(self):
         """
-        Test Timeout
+        Test from dark slit image to bright slit image (gray)
         """
         bl = self.light
-        ccd = mock.FakeCCD(0, self.img_light_off)
-        f = turnLightAndCheck(bl, ccd)
-        # no change in the image
+        ccd = mock.FakeCCD(self.img_ccd_loff)
+        f = turnOnLightAndCheck(bl, ccd)
         time.sleep(1)
         self.assertFalse(f.done())
-        # with self.assertRaises(TimeoutError):
-        #     f.result(timeout=5)
-        with self.assertRaises(futures.TimeoutError):
-            f.result(timeout=5)
-        f.cancel()
-        logging.warning("Light doesn't appear to turn on after 60s")
-
 
-    def test_slit_cancel(self):
-        """
-        Test cancelling does cancel (relatively quickly)
-        """
-        bl = self.light
-        ccd = mock.FakeCCD(0, self.img_light_off)
-        f = turnLightAndCheck(bl, ccd)
-        time.sleep(1)
-        f.cancel()
-        self.assertTrue(f.cancelled())
-        with self.assertRaises(CancelledError):
-            f.result(5)
+        # simulate light turning on
+        ccd.fake_img = self.img_ccd_lon
+        time.sleep(3)
+        self.assertTrue(f.done())
+        self.assertFalse(f.cancelled())
+        f.result()
 
-    def test_1pixel_image(self):
+    def test_1pix_off_on(self):
         """
         Test from dark image to bright image of 1 pixel
         """
         bl = self.light
-
-        #dark 1x1 image
-        img_dark = numpy.zeros((1,1), dtype= numpy.uint8)  # type: ndarray
+        img_dark = numpy.zeros((1, 1), dtype=numpy.uint8)  # type: ndarray
```

Comment:
I think this test case would be more realistic if the image had an initial value of 1, and would go to ~10 for the bright value.